addons/core/addon/styles/addon.scss

```diff
@@ -20,3 +20,47 @@
     }
   }
 }
+
+// Search & Filtering
```

Comment:
I moved this to core since I need these styles in admin anyways and this would help with merge conflicts later